bin/disco_vpc_ui.py

```diff
@@ -8,7 +8,11 @@
 import argparse
 import sys
 
-from disco_aws_automation import DiscoVPC
+from disco_aws_automation import (
```

Comment:
[**Nitpicking**] why not:

from disco_aws_automation import DiscoVPC, DiscoVPCPeerings

*Ref: https://www.python.org/dev/peps/pep-0008/#imports*